packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -173,7 +173,7 @@ WIZARD_NAV_ARIA_ROLE_DESCRIPTION = Wizard
 WIZARD_NAV_STEP_DEFAULT_HEADING=Step
 
 #XTIT: View Settings Dialog dialog title for Sort
-VSD_DIALOG_TITLE_SORT=Sort
+VSD_DIALOG_TITLE_SORT=View Settings
```

Comment:
Maybe VSD_DIALOG_TITLE only?

Reply:
This title is relevant only if the current mode is sort. If the mode changes, so does the title.